dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -372,10 +416,17 @@ public Response deleteBundlesOlderThan(@Context   final HttpServletRequest reque
 
             try {
 
-                final int bundleDeletedSize =
-                        this.bundleAPI.deleteBundleAndDependenciesOlderThan(olderThan, initData.getUser()).size();
+                final BundleDeleteResult bundleDeleteResult =
+                        this.bundleAPI.deleteBundleAndDependenciesOlderThan(olderThan, initData.getUser());
+
+                if (UtilMethods.isSet(bundleDeleteResult.getFailedBundleSet())) {
```

Comment:
This logic can be extracted to a new method as it appears at least 3 or 4 times on this class

Reply:
done